tests/main/canonical-livepatch/task.yaml

```diff
@@ -0,0 +1,19 @@
+summary: Ensure canonical-livepatch snap works
+
+# livepatch works only on 16.04 amd64 systems
+systems: [ubuntu-16.04-64]
+
+execute: |
+    echo "Ensure canonical-livepatch can be installed"
+    snap install canonical-livepatch
+
+    echo "Wait for it to respond"
+    for i in $(seq 30); do
+        if canonical-livepatch status; then
```

Comment:
Let's please send the stderr and out to &> /dev/null here, otherwise this will probably scroll a massive amount of garbage that we don't want to see in logs or send across for spread to cache.